INSTALL.md

```diff
@@ -86,44 +86,44 @@ helm init --upgrade --service-account tiller --wait
 CF-Operator can be installed in a separate namespace:
 
 ```
-helm install --namespace cfo --name cf-operator --set "operator.watchNamespace=scf" https://s3.amazonaws.com/cf-operators/helm-charts/cf-operator-v0.4.1%2B92.g77e53fda.tgz
+helm install --namespace cfo --name cf-operator --set "operator.watchNamespace=kubecf" https://s3.amazonaws.com/cf-operators/helm-charts/cf-operator-v0.4.1%2B92.g77e53fda.tgz
 ```
 
-This allows us to restart the operator, because it is not affected by webhooks. We can also delete the SCF deployment namespace to start from scratch, without redeploying the operator.
+This allows us to restart the operator, because it is not affected by webhooks. We can also delete the KUBECF deployment namespace to start from scratch, without redeploying the operator.
 
-## Install SCFv3
+## Install Kubecf
 
 Enable Eirini explicitly when installing. The `system_domain` DNS record needs to point to the IP of the external load balancer.
 
 ```
-helm install --namespace scf --name scf https://scf-v3.s3.amazonaws.com/scf-3.0.0-82165ef3.tgz --set "system_domain=scf.suse.dev" --set "features.eirini=true"
+helm install --namespace kubecf --name kubecf https://scf-v3.s3.amazonaws.com/scf-3.0.0-82165ef3.tgz --set "system_domain=kubecf.suse.dev" --set "features.eirini=true"
```

Comment:
Note that `kubecf.suse.dev` does not resolve.

```
$ dig scf.suse.dev
…
scf.suse.dev.           170     IN      A       192.168.77.77
…
$ dig kubecf.suse.dev
…
;; ->>HEADER<<- opcode: QUERY, status: NXDOMAIN, id: 19301
```

Reply:
Will ask @colstrom if that is fixable in our dns, i.e. possible to add the domain.